src/github/index.ts

```diff
@@ -49,6 +51,10 @@ export default (robot) => {
 		middleware(pullRequest)
 	);
 
+	robot.on("workflow_run", middleware(workflow));
+
+	robot.on("deployment_status", middleware(deployment));
```

Comment:
As per your changes above, shouldn't there be a `deployment` event as well?

Reply:
Nope, we only listen to `deployment_status`. I'll comment the `deployment` event from https://github.com/atlassian/github-for-jira/pull/573/files#diff-4f81d94913f59db9100bd467a208d1a30a2c9d548aa71d6a54e4fd710c3a3018R21